crates/rerun/src/run.rs

```diff
@@ -540,29 +593,30 @@ async fn run_impl(
                 server_memory_limit,
             )
             .await?;
-            let ws_server_url = ws_server.server_url();
+            let _ws_server_url = ws_server.server_url();
             let rx = ReceiveSet::new(rx);
             let ws_server_handle = tokio::spawn(ws_server.listen(rx));
 
-            // This is the server that serves the Wasm+HTML:
-            let web_server_handle = tokio::spawn(host_web_viewer(
-                args.bind.clone(),
-                args.web_viewer_port,
-                true,
-                ws_server_url,
-            ));
+            #[cfg(feature = "web_viewer")]
+            {
```

Comment:
```suggestion
            if cfg!(feature = "web_viewer") {

Reply:
Sadly no; we don't want to pull in the web viewer crate unless the feature is explicitly on